import-scripts/patients_vl.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import numpy as np
+import requests
+
+patients = np.genfromtxt('data/patients.csv', delimiter=',', dtype=None, encoding=None, skip_header=1)
+vl_obs = np.genfromtxt('data/vl_obs.csv', delimiter=',', dtype=None, encoding=None, skip_header=1)
```

Comment:
We'll probably need some sort of pre-processing before running this. Currently, the simulation outputs a CSV of visits per month which results in 72 files with ~28,000 obs per file. The simplest version of this is something like this:

```sh
cat *_visits.csv | awk 'NR > 1 {print}' > all-visits.csv
```

Which just creates a mammoth CSV of every data row from those visits. If loading an ~2 million line CSV works, we can just go with that. Otherwise, we may need to be more clever to, e.g., generate a CSV-per-patient and process that for every patient (we seem to be averaging ~14 obs per patient, so that definitely isn't terrible).